.cargo/config_apple_tvos

```diff
@@ -2,7 +2,7 @@
 [patch.crates-io]
 cc = { git = 'https://github.com/nordsecurity/cc-rs.git', branch = 'add_apple_tvos_support' }
 if-addrs = { git = 'https://github.com/nordsecurity/if-addrs.git', branch = 'add_apple_tvos_support_ifaddrs' }
-socket2 = { git = 'https://github.com/nordsecurity/socket2.git', branch = 'add_apple_tvos_support_0_4_9' }
+socket2 = { git = 'https://github.com/nordsecurity/socket2.git', branch = 'add_apple_tvos_support_0_4_10' }
```

Comment:
Maybe you could use some tag here? Using apple_tvos branch looks strange here IMHO

Reply:
Yeah, that is interesting leftover... But even if we were to update those, separate task makes more sense :P